ant/java.xml

```diff
@@ -251,7 +251,7 @@ your FindBugs installation's lib directory. E.g.:
       <doctitle><![CDATA[<h1>Bio-Formats</h1>]]></doctitle>
       <bottom><![CDATA[${copyright.begin} ${YEAR} ${copyright.end}]]></bottom>
       <link href="http://docs.oracle.com/javase/7/docs/api/"/>
-      <link href="http://rsbweb.nih.gov/ij/developer/api/"/>
+      <link href="https://imagej.nih.gov/ij/developer/api/"/>
```

Comment:
I get frequent timeouts with this URL, and Wayne himself reported that it can be unstable. How about using https://imagej.net/developer/api/ instead?